src/graphql/models/Category.js

```diff
@@ -32,33 +39,11 @@ const Category = new GraphQLObjectType({
         },
         ...pagingArgs,
       },
-      resolve({ id }, args, { userId, appId }) {
-        // TODO: Mock data, needs implementation
-        return [
-          {
-            aiModel: 'Model1',
-            aiConfidence: 0.8,
-            positiveFeedbackCount: 2,
-            negativeFeedbackCount: 1,
-            categoryId: id,
-            articleId: 'AVrwb4-OtKp96s659CvV',
-            status: 'NORMAL',
-            createdAt: '2020-02-06T05:34:45.862Z',
-            updatedAt: '2020-02-06T05:34:46.862Z',
-          },
-          {
-            // Simulate category that is added by current user
-            userId,
-            appId,
-            positiveFeedbackCount: 2,
-            negativeFeedbackCount: 1,
-            categoryId: id,
-            articleId: 'AWDuGBU6yCdS-nWhum2u',
-            status: 'NORMAL',
-            createdAt: '2020-02-06T05:34:45.862Z',
-            updatedAt: '2020-02-06T05:34:46.862Z',
-          },
-        ];
+      resolve: async ({ id }, { status }, { loaders }) => {
+        const articleCategories = await loaders.articleCategoriesByReplyIdLoader.load(
```

Comment:
I think the (not-yet implemented) dataloader should be `articleCategoriesByCategoryIdLoader` ? Since this is a field of `Category`.

Reply:
The implementation is in the PR 157. I will merge them later